pixleesdk/src/main/java/com/pixlee/pixleesdk/ui/adapter/PXLPhotoAdapter.kt

```diff
@@ -37,7 +45,7 @@ class PXLPhotoAdapter(
             is Item.Header -> TYPE_HEADER
             is Item.Content -> TYPE_ITEM
             is Item.LoadMore -> TYPE_LOAD_MORE
-
+            else -> TYPE_ITEM
```

Comment:
just sanity checking that this is correct syntax? i assume it is, ive just never seen it before ("is-else")

Reply:
[else is valid in Kotlin's pattern matching (`when`)](https://kotlinlang.org/docs/control-flow.html#when-expression), and can be used whether or not your cases use `is` or other conditions

Reply:
yes that's possible in Kotlin which is same as this below. This's kotlin's official syntax guide https://kotlinlang.org/docs/typecasts.html#unsafe-cast-operator
```kotlin
val item = list[getRealPosition(position)]
        return if(item is Item.Header) TYPE_HEADER
        else if(item is Item.Content) TYPE_ITEM
        else if(item is Item.LoadMore) TYPE_LOAD_MORE
        else TYPE_ITEM
```